jax/experimental/README.md

```diff
@@ -5,6 +5,10 @@ libraries are in part about providing tools and in part about serving as
 examples for how to build such libraries using JAX. Each one is only a few
 hundred lines of code, so take a look inside and adapt them as you need!
 
+#### 👉 **Note**: These mini-libraries are meant to be an _inspiration_, but not a prescription. 
+
+To serve that purpose, it is best to keep their code samples minimal; so we generally *will not merge PRs* adding new features. Instead, please send your lovely pull requests and design ideas to more fully-featured libraries like [DeepMind's Haiku](https://github.com/deepmind/dm-haiku) and Google Research's [FLAX](https://github.com/google/flax) or [Trax](https://github.com/google/trax)."
```

Comment:
This reminds me that we should mention these NN libraries in the main README. 

I would suggest dropping the "DeepMind" and "Google Research's" library attributions.

Reply:
SGTM! Dropping. :)